labgrid/strategy/bareboxstrategy.py

```diff
@@ -13,6 +13,7 @@ class Status(enum.Enum):
     unknown = 0
     barebox = 1
     shell = 2
+    off = 3
```

Comment:
Please keep the states in their "natural" order. Also sort the state handlers accordingly. So unknown, off, barebox, shell.